websites/B/Blinest/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "828916578698526722",
+		"name": "blinest#3458"
+	},
+	"service": "Blinest",
+	"description": {
+		"en": "Blinest is a free multiplayer music quiz site."
+	},
+	"url": "blinest.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/tAwvgnB.png",
```

Comment:
Logos should be a **square** with dimensions of [at least 512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).